### PR TITLE
1975: First commit with release docs. Need direction and review 

### PR DESCRIPTION
**Pull request by chanduvkp**
### 🛑 Important:
Please open an issue to discuss significant work before you start. We appreciate your contributions and don't want your efforts to go to waste!

## Purpose of this PR
This is to stream line release process for spark-operator 

**Proposed changes:**
- Added release directory and release.md file in it

## Change Category
Indicate the type of change by marking the applicable boxes:

- [X] Feature (non-breaking change which adds functionality)
- [X] Documentation update

### Rationale
Need direction on building the release process for spark operator

## Checklist
Before submitting your PR, please review the following:

- [x] I have conducted a self-review of my own code.
- [x] I have updated documentation accordingly.
- [x] I have added tests that prove my changes are effective or that my feature works.
- [x] Existing unit tests pass locally with my changes.

### Additional Notes
None

### CODE REVIEW

File: docs/release/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Prerequisite
+
+1. Permissions
+   - You need write permissions on the repository to create a release tag/branch.
+2. Prepare your Github Token
+
+
+### Release Process
+
+1. Make sure the last commit you want to release past `spark-operator -postsubmit` testing.
```
Comment (peter-mcclonski) [inline docs/release/README.md:12]: A: `past` -> `passed`

File: docs/release/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Releasing the Training Operator
```
Comment (ChenYi015) [inline docs/release/README.md:1]:
```suggestion
# Releasing the Spark Operator
```

**Comment by chanduvkp** [comment]
Thank you @ChenYi015 and @peter-mcclonski for the review. Updated the README.md file with the feedback

**Comment by ChenYi015** [comment]
@chanduvkp Thanks for the effort to add release docs! Since the release workflow and docs have already been updated, will close this as it has been out of date.

**Pull request closed without merging**